### ISSUE TITLE
rootless

### ISSUE THREAD

**Issue by TDPFlorida** [brendonjkding/VolumeMixer#8]
Is there a tweak in the future that will control all the different volumes on an iPhone that will work rootless?

**Comment by brendonjkding** [comment]
I'll release an update as soon as libcephei **really** supports rootless.

**Comment by TDPFlorida** [comment]
I’m not sure what you mean by that (I’m new to this whole rootless thing) but I here it is the new jailbreaking standard. And thanks for your speedy reply.

 

Tom

**Comment by Zhou-Shilin** [comment]
> I'll release an update as soon as libcephei **really** supports rootless.

May I ask what progress has been made so far?

**Comment by brendonjkding** [comment]
> > I'll release an update as soon as libcephei **really** supports rootless.
> 
> May I ask what progress has been made so far?

As far as I know, there's no update from libcephei.

But I do have some BAD news.

I recently added support for iOS 16 and during that I found C function hook is sort of broken on rootless for whatever reason, which means even libcephei is properly updated, apps using C-level volume API will not be supported until C function hook is fixed.

My personal recommendation is that non-developer should get rid of jailbreak as soon as possible. Jailbreak is dead.

**Comment by brendonjkding** [comment]
> The developer of libcephei said that it already support rootless with v2.0. Can you please check it?

Yes. v2.0 supports rootless, but just installable, otherwise it doesn't work. 

That's why I said:

> I'll release an update as soon as libcephei **really** supports rootless.

The word "really" matters.

**Comment by brendonjkding** [comment]
Rootless should now be supported in version 0.0.7~beta1.

<details>
<summary>P.S.</summary>
Almost two years have passed, and libcephei still not updated. What the ****???
</details>
